modules/sdk-coin-sui/src/lib/iface.ts

```diff
@@ -1,6 +1,30 @@
+import { TransactionExplanation as BaseTransactionExplanation, TransactionType } from '@bitgo/sdk-core';
+import { SuiObjectRef } from './transaction';
+
+export interface TransactionExplanation extends BaseTransactionExplanation {
+  type: TransactionType;
+}
+
+export interface PayTx {
```

Comment:
this is sui's internal representation, but it is hard to  understand, we can flatten it.